VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/video-endpoint-panel-model.ts

```diff
@@ -2,11 +2,11 @@ import { EndpointStatus, Role, VideoEndpointResponse } from '../../services/clie
 import { IndividualPanelModel } from './individual-panel-model';
 
 export class VideoEndpointPanelModel extends IndividualPanelModel {
-    public status: EndpointStatus;
+    public endpointStatus: EndpointStatus;
```

Comment:
Is this changed for a reason?

Reply:
no when I try to see learn the code I got confused with status

Reply:
I don't disagree, I just have a concern because I have seen places where a function will take any type and check ParticipantStatus or EndpointStatus as strings rather than using enums. I would hope the tests would catch something like that but I can't be sure.